tests/load/test_update_utils.py

```diff
@@ -1,5 +1,274 @@
+import unittest
+import pytest
+
 from scribe_data import utils
 
 
-def test_get_language_qid():
-    assert utils.get_language_qid("french") == "Q150"
+def test_get_scribe_languages():
+    test_case = unittest.TestCase()
+
+    # test for content, not order
+    test_case.assertCountEqual(
+        utils.get_scribe_languages(),
+        [
+            "English",
+            "French",
+            "German",
+            "Italian",
+            "Portuguese",
+            "Russian",
+            "Spanish",
+            "Swedish",
+        ],
+    )
+
+
+@pytest.mark.parametrize(
+    "language, qid_code",
+    [
+        ("English", "Q1860"),
+        ("french", "Q150"),
+        ("GERMAN", "Q188"),
+        ("iTalian", "Q652"),
+        ("poRTUGuese", "Q5146"),
+        ("russian", "Q7737"),
+        ("spanish", "Q1321"),
+        ("swedish", "Q9027"),
+    ],
+)
+def test_get_language_qid_positive(language, qid_code):
+    assert utils.get_language_qid(language) == qid_code
+
+
+def test_get_language_qid_negative():
+    with pytest.raises(ValueError) as excp:
+        _ = utils.get_language_qid("Newspeak")
```

Comment:
Nice easter eggs :laughing:

Reply:
Double plus good!

Please find enclosed `language_meta_data.json`.  The `description` & `used_by` fields are comments. The `languages` field is where the data are held proper. Not sure about the placement of the `remove` & `ignore` fields.

Writen a working implementation. Currently refactoring.

[language_meta_data.json.zip](https://github.com/scribe-org/Scribe-Data/files/12894951/language_meta_data.json.zip)

Reply:
Appreciate the easter egg as well! 🐰🥚😊 Till now we just have a random three emoji selection from "🥳🎉" as the gender annotation in the app when the user types "Scribe", and when you click the annotation it the selection is redone 🙃 It's fun 😇